compute_tools/src/config.rs

```diff
@@ -139,3 +142,38 @@ where
 
     res
 }
+
+/// Create or completely rewrite configuration file specified by `path`
+pub fn write_local_proxy_conf(path: &Path, local_proxy: &LocalProxySpec) -> Result<()> {
+    let config =
+        serde_json::to_string_pretty(local_proxy).context("serializing local-proxy json")?;
```

Comment:
nit: Technically, we don't know the name of the file. Maybe replace file name with `LocalProxySpec`.